src/Search/Kernel.php

```diff
@@ -434,8 +433,7 @@ public function dependencies(Application $app)
                 $app['console.gearman.task_driver'],
                 $app['elastic.client.write'],
                 $app['validator'],
-                $app['logger'],
-                $app['config']['feature_rds'] ? $app['config']['rds_articles'] : []
```

Comment:
The condition needs to be removed but the parameter is still expected. I will followup with a commit.